tests/unit/DataObjectAnnotatorTest.php

```diff
@@ -181,6 +236,73 @@ public function testInversePlayerRelationOfTeam()
         );
     }
 
+    public function testDefaults()
+    {
+        $count = 0;
+        DataObjectAnnotator::pushExtensionClass(Page::class);
+        foreach (DataObjectAnnotator::getExtensionClasses() as $class) {
+            if ($class === 'Page') {
+                $count++;
+            }
+        }
+        $this->assertEquals(1, $count);
+    }
+
+    public function testSetExtensionClasses()
+    {
+        $classes = DataObjectAnnotator::getExtensionClasses();
+        unset($classes[20]);
```

Comment:
Magic numbers! 😆

Reply:
I know, could not find a proper way around it when doing the initial work and left it to that :D